src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

```diff
@@ -6,12 +6,14 @@
 import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.drivers.SelectDriverMenuOptionListener;
 import edu.kis.powp.jobs2d.drivers.adapter.monitor.DeviceMonitorDriver;
+import edu.kis.powp.jobs2d.drivers.observer.ApplyDriverDecoratorsSubscriber;
 import edu.kis.powp.jobs2d.drivers.observer.UpdateDriverInfoSubscriber;
 
 public class DriverFeature {
 
     private static DriverManager driverManager = new DriverManager();
     private static Application app;
+    private static DeviceMonitorDriver deviceMonitorDriver = new DeviceMonitorDriver();
```

Comment:
DeviceMonitorDriver should be a specific configurable feature. Please move this dependency to a separate unit (e.g. UsageMonitorFeature).

Reply:
The dependency is still there! The goal was to remove any references to usage monitoring from this class and move them e.g. to UsageMonitorFeature.